packages/core/kind/kind.js

```diff
@@ -135,7 +136,7 @@ const kind = (config) => {
 
 		static propTypes = propTypes
 
-		static contextTypes = contextTypes
```

Comment:
We can't remove `contextTypes` yet (it's only deprecated) but I just verified that if both are present that `contextType` will take precedence so we can leave the both for now and remove `contextTypes` fully in 3.0.

Reply:
Sandbox link to try them out together if you're interested: https://codesandbox.io/s/o53r87r9w9